client/main.scss

```diff
@@ -51,6 +51,9 @@ html, body, #react-root, [data-reactroot] {
     position: relative;
     overflow-y: auto;
     flex: 1 1;
+    .container {
+      padding-bottom: 1rem;
+    }
```

Comment:
Not necessarily on this element, maybe on a containing one if it avoids duplication.